### DISCUSSION TITLE
add custom css

### DISCUSSION THREAD

**Discussion by unonweb** [payloadcms/payload#1514] [Q&A] [answered]
Hello there,

I've tried to add my custom css according to the [docs](https://payloadcms.com/docs/admin/customizing-css#customizing-css-and-scss).
But in the end the stylesheet isn't included (neither as inline nor as external). Does it work for you? Is there anything I need to consider?
In the docs it looks very simple...

**Comment by JessRynkar** [comment, marked as answer]
Hi @unonweb,

This could be a cache issue - could you try and run `rm -rf node_modules/.cache` then restart your server and let me know how that goes.

**Comment by mvdve** [comment]
Cache was indeed the issue.
